src/index.css

```
@@ -7,7 +7,7 @@
 @tailwind utilities;
 
 :root {
-  --infra-font-default: HelsinkiGrotesk, Arial, sans-serif;
+  --infra-font-default: HelsinkiGrotesk Arial, sans-serif;
```

A comma is missing after HelsinkiGrotesk. Without this, it uses `sans-serif`.
```suggestion
  --infra-font-default: HelsinkiGrotesk, Arial, sans-serif;
```